src/app/features/exchange-set/ess-add-single-encs/ess-add-single-encs.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="container">
```

Comment:
This file is out of date with the 13016 branch

Reply:
I have merged 13016 branch into my branch so that I can use file Upload control.
once 13016 Branch is merged into main, it will be in sync.